sale_service_fleet/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+Sale Service Fleet
+==================
+
+This module was written to extend the functionality of sale service project
+and allow you to assign a vehicle to a sale order, so when the project is
+created the vehicle is assigned to it.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a vehicle in fleet menu and assign it to a sale order.
```

Comment:
I see in the code many more functionalities. Could you explain yourself better? Just to know what am I supposed to test in the runbot.